test/interactions/clickInteractionTests.ts

```diff
@@ -2,156 +2,186 @@
 
 describe("Interactions", () => {
   describe("Click Interaction", () => {
```

Comment:
Although possibly it would be better to have only the class name, so that the error messages would read "`Interactions Click ...`" instead of "`Interactions Click Interaction ... `".